freegs4e/critical.py

```diff
@@ -599,8 +599,14 @@ def core_mask(R, Z, psi, opoint, xpoint=[], psi_bndry=None):
 
 
 @njit(fastmath=True, cache=True)
-def inside_mask(
-    R, Z, psi, opoint, xpoint=[], mask_outside_limiter=None, psi_bndry=None
+def inside_mask_(
+    R,
+    Z,
+    psi,
+    opoint,
+    xpoint=[],
+    mask_outside_limiter=None,
+    psi_bndry=None,
 ):
     """
     Similar to core_mask_old above, except:
```

Comment:
The `core_mask_old` has since been removed. Could this `inside_mask_` function be documented?

Reply:
Resolved in #12